packages/web/src/hooks/useAdminUpdateUser.js

```diff
@@ -0,0 +1,34 @@
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+import api from 'helpers/api';
+import useEnqueueSnackbar from 'hooks/useEnqueueSnackbar';
+import useFormatMessage from 'hooks/useFormatMessage';
+
+export default function useAdminUpdateRole(userId) {
```

Comment:
I believe the function name here is incorrect, since the file name is useAdminUpdateUser.